### PR TITLE
added do-not-disturb.md

### PR DESCRIPTION
**Pull request by andreas-kater**

### CODE REVIEW

File: handbook/README.md [hunk 1]
```diff
@@ -10,8 +10,9 @@ This policy does not form part of any employee&#39;s contract of employment and
   * [Using the Staff Handbook](#using-the-staff-handbook)
   * [Responsibility for the Staff Handbook](#responsibility-for-the-staff-handbook)
 * Best Practice
-  * [Anti-corruption and Bribery Policy](best-practice/anti-corruption.md)
-  * [Anti-harassment and Bullying Policy](best-practice/anti-harassment.md)
+  * [Anti-Corruption and Bribery Policy](best-practice/anti-corruption.md)
+  * [Anti-Harassment and Bullying Policy](best-practice/anti-harassment.md)
+  * [Do-Not-Disturb Mode](best-practice/anti-harassment.md)
```
Comment (tom-mullen) [inline handbook/README.md]: Feels like this one should be in the quick reference guide?

File: handbook/README.md [hunk 2]
```diff
@@ -10,8 +10,9 @@ This policy does not form part of any employee&#39;s contract of employment and
   * [Using the Staff Handbook](#using-the-staff-handbook)
   * [Responsibility for the Staff Handbook](#responsibility-for-the-staff-handbook)
 * Best Practice
-  * [Anti-corruption and Bribery Policy](best-practice/anti-corruption.md)
-  * [Anti-harassment and Bullying Policy](best-practice/anti-harassment.md)
+  * [Anti-Corruption and Bribery Policy](best-practice/anti-corruption.md)
+  * [Anti-Harassment and Bullying Policy](best-practice/anti-harassment.md)
```
Comment (tom-mullen) [inline handbook/README.md]: These look like they belong in a different PR?

File: handbook/best-practice/do-not-disturb.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+The way we signal to the rest of the team that we are in such a focus session is
+by turning on Slack's `Do not disturb` mode.
+
+When this status is turned on, people who try to send us a Slack message receive
+a notification saying we are in `Do not disturb` mode and asking whether they
+want to send a notification despite this.
+
+In the rare circumstances where a pressing urgent matter needs immediate
+attention, we ask for a notification to be sent anyway. Otherwise we make a note
```
Comment (tom-mullen) [inline handbook/best-practice/do-not-disturb.md]: I personally use `/remind me to message this person in 1 hour` functionality in slack as a reminder.

**Review by tom-mullen — CHANGES_REQUESTED** [review]

**Review by annisem — APPROVED** [review]
Looks good

**Pull request merged**